app/build.gradle.kts

```diff
@@ -40,10 +41,14 @@ android {
             )
 
             signingConfig = signingConfigs.getByName("release")
+
+            buildConfigField(BuildConfigFields.API_KEY.type, BuildConfigFields.API_KEY.title, BuildConfigFields.API_KEY.value)
```

Comment:
Why do you need to add this line? as compared to line 51 ?

Reply:
I don't need line 45 as i am adding API_KEY for all build types. Removed.